src/pages/edge/adobe-journey-optimizer-decisioning/optimize-configuration-view.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Decisioning Validation view
+description: An overview of how the Optimize datastream and sdk setup can be validated.
+keywords:
+- Datastream Validation
+- SDK Validation
+---
+
+# Decisioning Validation view
+
+## Overview
+
+You can use the Decisioning Validation view to check if your mobile application has been set up successfully to use Adobe Experience Platform Optimize SDK. The view validates all pre-requisites have been met and offers feedback on how to fix your setup in cases where requirements have been missed.
+
+## Getting started
+
+To use this view, follow the steps outlined below:
+
+1. [Setup](./assurance-setup.md) an Assurance session
+2. In the **Assurance Session home** view, select **Validate Setup**
```

Comment:
What is `Assurance Session home view`? Do you mean the Assurance plugins section on the left hand side?